### PR TITLE
Update publications.yml

### PR DESCRIPTION
**Pull request by romain-lopez**
added scanvi

### CODE REVIEW

File: _data/publications.yml [hunk 1]
```diff
@@ -156,7 +156,7 @@
 
 - authors: "C. Xu\\*, R. Lopez\\*, E. Mehlman\\*, J. Regier, M.I. Jordan, N. Yosef"
   title: "Probabilistic Harmonization and Annotation of Single-cell Transcriptomics data with Deep Generative Models."
-  details: bioRxiv
+  details: Molecular Systems Biology (in press)
```
Comment (adamgayoso) [inline _data/publications.yml]: @romain-lopez can you remove the preprint flag for this entry?

**Review by adamgayoso — CHANGES_REQUESTED** [review]

File: _data/publications.yml (hunk 1 above)
Comment (romain-lopez) [reply _data/publications.yml]: ah yes!

File: _data/publications.yml [hunk 2]
```diff
@@ -226,6 +123,107 @@
...
+  github: https://github.com/PierreBoyeau/sbVAE
+  arxiv: https://arxiv.org/abs/2002.07217
+
+- authors: "R. Lopez, A. Gayoso & N. Yosef."
+  title: "Enhancing Scientific Discoveries in Molecular Biology with Deep Generative Models."
+  year: 2020
+  doi: 10.15252/msb.20199198
+  details: Molecular Systems Biology
+  
+  - authors: "C. Xu\\*, R. Lopez\\*, E. Mehlman\\*, J. Regier, M.I. Jordan, N. Yosef"
```
Comment (adamgayoso) [inline _data/publications.yml]: I think there's a spacing issue here

**Review by adamgayoso — CHANGES_REQUESTED** [review]

File: _data/publications.yml (hunk 2 above)
Comment (adamgayoso) [inline _data/publications.yml]:
```suggestion
- authors: "C. Xu\\*, R. Lopez\\*, E. Mehlman\\*, J. Regier, M.I. Jordan, N. Yosef"
```

**Review by adamgayoso — CHANGES_REQUESTED** [review]

**Pull request merged**